spring/spring-boot-maven-mysql/.platform/local/.gitignore

```diff
@@ -0,0 +1 @@
+/
```

Comment:
Huh.  The .platform/local directory shouldn't even be commitable.  It should be excluded by `.gitignore`.  I'm not even sure how you got these in there. :smile: Remove please.